### PR TITLE
Update the ResidentialLightingOther schedule in the options_lookup.tsv

### PR DESCRIPTION
**Pull request by mdahlhausen**
The old schedules were from the T24 2016 Residential ACM, except they were using the residential MELS schedule instead of the exterior lighting schedule.  This commit updates to use the properly intended exterior lighting schedule instead.

See the T24 2016 Residential ACM and the Plug Loads and Lighting Modeling CASE report from Energy Solutions for more information.

Addresses #[issue number here].

## Pull Request Description

[description here]

## Checklist

Not all may apply:

- [ ] Unit tests have been added or updated
- [x] All rake tasks have been run, and pass
- [ ] Documentation has been modified appropriately
- [ ] Any new options are added to `project_testing`
- [ ] `project_testing` runs without any failures
- [ ] No unexpected regression test changes
- [x] All tests are passing (green) on circleci
- [x] The [changelog](https://github.com/NREL/OpenStudio-BuildStock/blob/master/CHANGELOG.md) has been updated appropriately
- [x] This branch is up-to-date with master

For more information on how to perform these checklist items, see the documentation's [Advanced Tutorial](https://resstock.readthedocs.io/en/latest/advanced_tutorial/index.html).

### CODE REVIEW

File: resources/options_lookup.tsv [hunk 1]
```diff
@@ -1756,23 +1756,23 @@ Clothes Dryer	None	ResidentialApplianceClothesDryer	fuel_type=electric	cef=2.70
...
+		ResidentialLightingOther	hw_cfl=0	hw_led=0	hw_lfl=0	pg_cfl=0	pg_led=0	pg_lfl=0	in_eff=15	cfl_eff=55	led_eff=80	lfl_eff=88	option_type=Lamp Fractions	energy_use_garage=100	energy_use_exterior=300	sch_option_type=Calculated Lighting Schedule	"weekday_sch=0.046, 0.046, 0.045, 0.045, 0.046, 0.045, 0.044, 0.041, 0.036, 0.03, 0.024, 0.016, 0.012, 0.011, 0.011, 0.012, 0.019, 0.038, 0.048, 0.06, 0.083, 0.098, 0.085, 0.059"	"weekend_sch=0.046, 0.046, 0.045, 0.045, 0.046, 0.045, 0.044, 0.041, 0.036, 0.03, 0.024, 0.016, 0.012, 0.011, 0.011, 0.012, 0.019, 0.038, 0.048, 0.06, 0.083, 0.098, 0.085, 0.059"	"monthly_sch=1.248, 1.257, 0.993, 0.989, 0.993, 0.827, 0.821, 0.821, 0.827, 0.99, 0.987, 1.248"
```
Comment (ejhw) [inline resources/options_lookup.tsv]: @mdahlhausen For this schedule to actually get used, you need to change `sch_option_type` to "User-Specified Lighting Schedule"

Comment (mdahlhausen) [reply resources/options_lookup.tsv]: Good catch. I've updated to use the User-Specified Lighting Schedule.  At some point in the future someone can make a calculated version for exterior lights.

File: resources/measures/ResidentialLightingOther/measure.rb [hunk 2]
```diff
@@ -157,14 +157,14 @@ def arguments(model)
     weekday_sch = OpenStudio::Measure::OSArgument::makeStringArgument("weekday_sch", true)
     weekday_sch.setDisplayName("Weekday schedule")
     weekday_sch.setDescription("Specify the 24-hour weekday schedule.")
-    weekday_sch.setDefaultValue("0.035, 0.033, 0.032, 0.031, 0.032, 0.033, 0.037, 0.042, 0.043, 0.043, 0.043, 0.044, 0.045, 0.045, 0.044, 0.046, 0.048, 0.052, 0.053, 0.05, 0.047, 0.045, 0.04, 0.036")
+    weekday_sch.setDefaultValue("0.046, 0.046, 0.045, 0.045, 0.046, 0.045, 0.044, 0.041, 0.036, 0.03, 0.024, 0.016, 0.012, 0.011, 0.011, 0.012, 0.019, 0.038, 0.048, 0.06, 0.083, 0.098, 0.085, 0.059")
```
Comment (ejhw) [inline resources/measures/ResidentialLightingOther/measure.rb]: Please add comment documenting origin of this schedule

Comment (mdahlhausen) [reply resources/measures/ResidentialLightingOther/measure.rb]: done.  Also realized that weekday and weekend schedules were slightly different so I updated the options_lookup.tsv accordingly.

**Comment by ejhw** [comment]
@mdahlhausen looks like this still needs to be added to the changelog. Something like "Exterior lighting schedule changed from using interior lighting sunrise/sunset algorithm to T24 2016 Residential ACM Appendix C Table 8 Exterior Lighting Hourly Multiplier for weekdays and weekends".

**Comment by mdahlhausen** [comment]
@ejhw added to the log and resolved merge conflict with master (log and options_lookup.tsv)

**Pull request merged**